content/docs/user-guide/troubleshooting.md

```diff
@@ -53,3 +53,16 @@ Unable to detect supported link types, as the
 [cache directory](/doc/command-reference/config#cache) doesn't exist. It is
 usually created automatically by DVC commands that need it, but you can create
 it manually (e.g. `mkdir .dvc/cache`) to enable this check.
+
+## Checkout failed with `store: false` outputs {#checkout-no-store}
+
+Unable to checkout this output since it is not present in the cache and has been
+marked to not be stored on remotes.
+
+If `store: false` is added to the `outs` section of a `.dvc` file, that output
+will not be stored on remotes when `dvc push` is used, and dvc will not attempt
+to retreive it when `dvc pull` or `dvc fetch` is called. This error means that
```

Comment:
```suggestion
DVC is unable to checkout any output that is not present in the
<abbr>cache</abbr>.This can usually be solved by fetching from
[import storage](/doc/command-reference/remote). However, If `store: false`
is used in a `.dvc` file, `dvc pull` and `dvc fetch` won't attempt to retrieve the
data from a DVC remote.

This error means that
```